Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_ctf.nut

```diff
@@ -408,7 +412,7 @@ void function CaptureFlag( entity player, entity flag )
 	AddTeamScore( team, 1 )
 	AddPlayerScore( player, "FlagCapture", player )
 	player.AddToPlayerGameStat( PGS_ASSAULT_SCORE, 1 ) // add 1 to captures on scoreboard
-	SetRoundWinningKillReplayAttacker( player ) // set attacker for last cap replay
+	//SetRoundWinningKillReplayAttacker( player ) // set attacker for last cap replay
```

Comment:
Is there a reason for this?

Reply:
i'll merge if this is figured out @DBmaoha, from my perspective this looks like it'd break last cap replay

Reply:
someting I modified for my own use, forgot to reset it before pr